dotnet/src/Functions/Functions.UnitTests/OpenAPI/RestApiOperationRunnerTests.cs

```diff
@@ -1009,6 +1010,77 @@ public async Task ItShouldThrowExceptionForUnsupportedContentTypeAsync()
         await Assert.ThrowsAsync<SKException>(() => sut.RunAsync(operation, arguments));
     }
 
+    public class SchemaTestData : IEnumerable<object[]>
+    {
+        public IEnumerator<object[]> GetEnumerator()
+        {
+            yield return new object[] {
+                    "default",
+                    new (string, RestApiOperationResponse)[] {
+                        ("400", new RestApiOperationResponse("fake-content", "fake-content-type", JsonDocument.Parse("{\"title\":\"FakeResponse\",\"type\":\"object\",\"properties\":{\"fakeItems\":{\"type\":\"array\",\"items\":{\"title\":\"Item\",\"type\":\"object\",\"properties\":{\"attributes\":{\"type\":\"array\",\"itemName\":{\"type\":\"string\"}},\"name\":{\"type\":\"string\"}}}}}}"))),
+                        ("default", new RestApiOperationResponse("Default response content", "application/json", JsonDocument.Parse("{\"title\":\"DefaultResponse\",\"type\":\"object\",\"properties\":{\"code\":{\"type\":\"integer\",\"format\":\"int32\"},\"message\":{\"type\":\"string\"}}}"))),
+                    },
+            };
```

Comment:
This code is read-only, making it very difficult to support in the future if any adjustments are required. Consider either to read JSONs from resource file(s) or build C# object and serialize it to JSON.